paper-checkbox.html

```diff
@@ -187,6 +187,7 @@
         white-space: normal;
         line-height: normal;
         color: var(--paper-checkbox-label-color, var(--primary-text-color));
+        width: calc(100% - var(--paper-checkbox-size) - var(--paper-checkbox-label-spacing));
```

Comment:
Hmm, slight hiccup here: this change prevents you from using `0` for `--paper-checkbox-label-spacing` if you're trying to clear the styling so you can use an external label, which I suspect is a relatively common case. Particularly, you can't add lengths (which have units) and unitless values with `calc()`. Also, you can't multiply two or more values having units, as the result would have units which aren't lengths; so multiplying by `1px` or something to get around this is out. :/